libcfnet/protocol.h

```diff
@@ -64,4 +64,17 @@
  */
 Seq *ProtocolOpenDir(AgentConnection *conn, const char *path);
 
+/**
+ * Receives statistics about a remote file.
+ *
+ * For the time being this is only a wrapper for #cf_remote_stat
```

Comment:
Is it?

Reply:
Nope, left over from its previous iteration. :^)